1-JavaScript/Week3/homework/ex1-giveCompliment.js

```diff
@@ -17,12 +17,26 @@
    Use `console.log` each time to display the return value of the
    `giveCompliment` function to the console.
 -----------------------------------------------------------------------------*/
-function giveCompliment(/* TODO parameter(s) go here */) {
+function giveCompliment(/* TODO parameter(s) go here */ name) {
```

Comment:
Please remove the instructions and TODO's if you've implemented them. 
```suggestion
function giveCompliment(name) {
```